config_defaults_inc.php

```diff
@@ -2085,6 +2085,28 @@
 $g_bug_closed_status_threshold = CLOSED;
 
 /**
+ * Bug becomes closed for submission of automated bug reports (attaches)
+ * if its status is >= $g_bug_collect_reports_threshold. The bug starts 
+ * accept more bug reports again if its status becomes less than this 
+ * threshold. The default is ASSIGNED. 
+ * Read-only, closed and fixed bugs are closed for bug report submission 
+ * regardless of this option.
+ * See also $g_bug_max_auto_reports option.
+ * See also SOAP.mc_issue_report function.
+ * @global integer $g_bug_collect_reports_status_threshold
+ */
+$g_bug_collect_reports_status_threshold = ACKNOWLEDGED;
+
+/**
+ * Maximum number of automatic reports to collect per bug when bug is open
+ * for collection of new reports (e.g. bug is not read-only, not closed, 
+ * not fixed, and not switched to $g_bug_collect_reports_status_threshold state). 
+ * See also SOAP.mc_issue_report function.
+ * @global integer $g_bug_max_auto_reports
+ */
+$g_bug_max_auto_reports = 50;
```

Comment:
It would be useful to have all crash reports tagged with a tag like 'crash', this way it is easy to filter on such issues.  This can be part of the issue information provided to issue report or we can add a configuration option to auto-add this to all issues added through such APIs whether or not the app provides the tag.